tortoise/__init__.py

```diff
@@ -158,14 +158,21 @@ def describe_field(name: str) -> dict:
             }
 
             # Foreign Keys have
-            if isinstance(field, fields.ForeignKeyField):
+            if isinstance(field, (fields.ForeignKeyField, fields.OneToOneField)):
                 del desc["db_column"]
                 desc["raw_field"] = field.source_field
             else:
                 del desc["raw_field"]
 
             # These fields are entierly "virtual", so no direct DB representation
-            if isinstance(field, (fields.ManyToManyFieldInstance, fields.BackwardFKRelation)):
+            if isinstance(
+                field,
+                (
+                    fields.ManyToManyFieldInstance,
+                    fields.BackwardFKRelation,
+                    fields.BackwardOneToOneRelation,
+                ),
+            ):
```

Comment:
Should probably extend the field definition by addind a "o2o_fields" list instead. o2o may be implemented much like foreign keys, but are conceptually different. So lets keep them separate in `describe_model`